utility/types/proto/actor.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
```

Comment:
The comments were resolved but not tended to?

<img width="1476" alt="Screen Shot 2022-09-29 at 1 57 00 PM" src="https://user-images.githubusercontent.com/1892194/193140523-374361c5-debc-44d5-a2d3-4adff9c1591d.png">